server/router.go

```diff
@@ -6,31 +6,30 @@ import (
 )
 
 func setupRoutes(router *gin.Engine) {
-	// v1 := router.Group("/api/v1")
-	client := Connect2DB()
+	dbClient := Connect2DB()
 	router.GET("/", hello)
-	router.GET("/login", loginPage)
-	router.GET("/list", attachDB(client, listPage))
+	router.GET("/login", loginPage)                   // TODO: not used
+	router.GET("/list", attachDB(dbClient, listPage)) // TODO: not used
 
-	router.GET("/products", attachDB(client, getAllProductsHandler))
-	router.GET("/products/:id", attachDB(client, getProductByIDHandler))
-	router.GET("/products-user/:id", attachDB(client, getProductByUserIDHandler))
```

Comment:
dont we need this?

Reply:
we need, and I have change the url of them,
for example, the third one -> /users/:id/products

Reply:
cool